index.html

```diff
@@ -1169,19 +1169,77 @@ <h2>Relative DID URLs</h2>
+independent of any specific representation. This section provides a
+high-level description of the data model, a set of requirements for
+representations, and a set of requirements for extensibility.
+    </p>
     <section>
       <h2>Definition</h2>
+      <p>
+A <a>DID document</a> consists of a collection of name-value pairs which will be
+referred to as properties in this document. The definitions of each of these
+properties are specified in section <a href="#core-properties"></a>. Specific
+representations are defined in section <a href="#core-representations"></a>.
+      </p>
     </section>
-
     <section>
-      <h2>Extensibility</h2>
+      <h2>Representations</h2>
+      <p>
+Following are the requirements for conformant representations.
+      </p>
+      <ol>
+        <li>
+A representation MUST define an unambiguous encoding and decoding of all
+properties defined in this specification. This definition MUST enable lossless
+conversion of the value of any property into another representation. This means
+anything you can represent in the DID document model can be represented in any
+compliant representation.
+        </li>
+        <li>
+The representation MUST have an unambiguous MIME type.
+        </li>
+        <li>
+The representation MUST define fragment processing rules for its MIME type that
+are conformant with the fragment processing rules defined in section
+<a href="#fragment"></a> of this specification.
+        </li>
+      </ol>
+        <p>
+The core representations are specified in section <a href="#core-representations"></a>.
+        </p>
     </section>
-
     <section>
-      <h2>Representation Requirements</h2>
+      <h2>Extensibility</h2>
+      <p>
+The data model supports two types of extensibility.
+      </p>
+      <ol>
+        <li>
+For maximum interoperability, it is RECOMMENDED that extensions use the official
+W3C registries defined by the DID Core and Extensions Registries specification
+[[DID-CORE-REGISTRIES]]. The use of this registry for new properties or other
+extensions is the only specified method for two independent implementations to
+be able to work together. Representations MUST process any extension in the DID
+Core and Extensions Registries according to the definition in the registry.
+Registry-based extensions MUST enable lossless conversion between all conformant
+representations.
+        </li>
+      <li>
+Representations MAY define other extensibility mechanisms including methods for
+decentralized extensions. Such extension mechanisms SHOULD support lossless
+conversion into any other conformant representation.
+      </li>
+      </ol>
+      <p class="note">
+It is always possible for two specific implementations to agree out-of-band to
+use a shared extension property or a new representation that is not recorded in
+the [[DID-CORE-REGISTRIES]]. However interoperability between such
+implementations may be less reliable.
```

Comment:
```suggestion
It is always possible for two specific implementations to agree out-of-band to
use a mutually understood extension or representation that is not recorded in
the DID Core Registries [[DID-CORE-REGISTRIES]]; interoperability between such
implementations and the larger ecosystem will be less reliable.
```

Removing MAY language so it's not confused to be a normative statement.

Reply:
Good suggestion.